ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/image/v1/image/image.html

```diff
@@ -0,0 +1,48 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div data-sly-use.image="com.adobe.cq.forms.core.components.models.form.StaticImage"
+     data-sly-use.formstructparser="com.adobe.cq.forms.core.components.models.form.FormStructureParser"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     data-sly-test="${image.imageSrc}"
+     data-cmp-src="${image.imageSrc}"
+     data-asset-id="${image.uuid}"
+     id="${image.id}"
+     data-cmp-data-layer="${image.data.json}"
+     data-cmp-hook-image="imageV3"
+     data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}"
+     data-cmp-is="adaptiveFormImage"
+     class="cmp-image${!wcmmode.disabled ? ' cq-dd-image' : ''}">
+    <a data-sly-unwrap="${!image.imageLink.valid}"
+       class="cmp-image__link"
+       data-sly-attribute="${image.imageLink.htmlAttributes}"
+       data-cmp-clickable="${image.data ? true : false}">
+        <img src="${image.imageSrc}"
+             class="cmp-adaptiveform-image__widget cmp-adaptiveform-image__image"
```

Comment:
this is not a widget. A widget is basically an element with which user can interact with